.gitignore

```diff
@@ -38,5 +38,8 @@ pip-log.txt
 sdist/
 target/
 var/
+local/
 node_modules
 .DS_Store
+Makefile
+manage.py
```

Comment:
Just curious. What are these used for?

Btw. you can use `.git/info/exclude` for your personal ignores.